internal/apiconfig/database_provider.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2020 Google LLC
```

Comment:
@mikehelmick since the loading of the configs is cached, do you think we should remove caching from the secret manager calls?